starlark with aggregation trigger goroutine panic

### Relevant telegraf.conf:
- `telegraf.conf`
```toml
[agent]

[[inputs.file]]
  files = ["metrics_in_1.json"]
  data_format = "json"
  json_time_format = "unix"

[[processors.starlark]]
  order = 1
  source = '''
def apply(metric):
  metric.fields["delay"] = int(metric.time / 1000000000) - (metric.fields["time"] / 1000)
  metric.fields.pop("time")
  metric.fields.pop("timeout")
  return metric
'''

[[aggregators.minmax]]
  period = "60s"
  drop_original = true

[[outputs.file]]
  files = ["stdout", "metrics_delay_max.out"]
```
- `metrics_in_1.json`
```json
{"time": 1594971809000, "timeout": 100}
```

### System info:
- Telegraf Version: `telegraf-1.15.0~rc3`
- Operating System: `Mac OS 10.15.5 (19F101)`


<!-- Include Telegraf version, operating system, and other relevant details -->

### Docker
- X

### Steps to reproduce:
- `$ telegraf --config telegraf.conf`


### Expected behavior:
- processor: `starlark`  + aggregator: `minmax` => Aggregation Success

### Actual behavior:
- Error msg
```
2020-07-17T07:52:40Z D! [aggregators.minmax] Updated aggregation range [2020-07-17 16:52:40 +0900 KST, 2020-07-17 16:52:50 +0900 KST]
panic: runtime error: index out of range [0] with length 0

goroutine 12 [running]:
github.com/influxdata/telegraf/plugins/processors/starlark.(*Starlark).Add(0xc0005d5c00, 0x2f58260, 0xc0008ce380, 0x2f49200, 0xc00020a060, 0x0, 0x0)
/go/src/github.com/influxdata/telegraf/plugins/processors/starlark/starlark.go:127 +0xa4a
github.com/influxdata/telegraf/models.(*RunningProcessor).Add(0xc0000a5a70, 0x2f58260, 0xc0008ce380, 0x2f49200, 0xc00020a060, 0x0, 0x0)
/go/src/github.com/influxdata/telegraf/models/running_processor.go:95 +0x106
github.com/influxdata/telegraf/agent.(*Agent).runProcessors.func1(0xc0005d0000, 0xc0001883e0)
/go/src/github.com/influxdata/telegraf/agent/agent.go:541 +0x156
created by github.com/influxdata/telegraf/agent.(*Agent).runProcessors
/go/src/github.com/influxdata/telegraf/agent/agent.go:536 +0x90
```

### Additional info:

<!-- Include gist of relevant config, logs, etc. -->

--- a/agent/agent.go
+++ b/agent/agent.go
@@ -224,6 +224,13 @@ func (a *Agent) initPlugins() error {
 				aggregator.Config.Name, err)
 		}
 	}
+	for _, processor := range a.Config.AggProcessors {
+		err := processor.Init()
+		if err != nil {
+			return fmt.Errorf("could not initialize processor %s: %v",
+				processor.Config.Name, err)
+		}
+	}
 	for _, output := range a.Config.Outputs {
 		err := output.Init()
 		if err != nil {

